test/windows/checkout/index.js

```diff
@@ -55,6 +55,22 @@ if (action === 'checkout') {
         });
     });
 
+} else if (action === 'shippingChange') {
+
+    window.xprops.payment().then(paymentToken => {
+
+        window.xprops.onShippingChange({
+            paymentID:    paymentToken,
+            city:         'XXXXX',
+            state:        'YY',
+            postal_code:  '11111',
+            country_code: 'YY'
+        }, {
+            reject: (callback) => {
```

Comment:
I don't see anyone passing this callback?

Reply:
Hmm perhaps there's a better way to test this, but I'm just trying to assert that the action can be passed up from the child and called in the parent.